src/net/l0/net_rx.c

```diff
@@ -33,7 +33,7 @@ int net_rx(struct sk_buff *skb) {
 	assert(skb != NULL);
 	assert(skb->dev != NULL);
 	if (skb->len < skb->dev->hdr_len) {
-		DBG(printk("net_rx: %p invalid length %zu\n", skb,
+		DBG(log_error("net_rx: %p invalid length %zu\n", skb,
```

Comment:
Regarding the code: shouldn't all these occurrences (`DBG(log_xxx(..))`) be changed to a plain `log_debug` call? That is, to get rid of these nasty conditional macros eventually. @anton-bondarev what do you think?